docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1305 @@
+{
```

Comment:
<p>Why do we need that here?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
I removed the Maximum Likelihood section as discussed.